### PR TITLE
Update vmware_guest_disk_info.py

### PR DESCRIPTION
**Pull request by yuramat**
##### SUMMARY

For returning value: "backing_diskmode".

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by goneri** [comment]
recheck

**Comment by yuramat** [comment]
recheck

**Comment by goneri** [comment]
recheck

**Comment by yuramat** [comment]
recheck

**Review by goneri — APPROVED** [review]

**Comment by goneri** [comment]
Looks good to me, thanks @yuramat for the contribution. Could you please just add a changelog entry:
https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#changelogs

File: lib/ansible/modules/cloud/vmware/vmware_guest_disk_info.py [hunk 1]
```diff
@@ -230,6 +230,7 @@ def gather_disk_info(self, vm_obj):
                     disks_info[disk_index]['backing_writethrough'] = bool(disk.backing.writeThrough)
                     disks_info[disk_index]['backing_thinprovisioned'] = bool(disk.backing.thinProvisioned)
                     disks_info[disk_index]['backing_eagerlyscrub'] = bool(disk.backing.eagerlyScrub)
+                    disks_info[disk_index]['backing_diskmode'] = disk.backing.diskMode
```
Comment (Akasurde) [inline lib/ansible/modules/cloud/vmware/vmware_guest_disk_info.py:233]:
```suggestion
                    disks_info[disk_index]['backing_diskmode'] = disk.backing.diskMode
                    disks_info[disk_index]['backing_disk_mode'] = disk.backing.diskMode
```

For backward compatability.

**Review by Akasurde — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/vmware/vmware_guest_disk_info.py (hunk 1 above)
Comment (Akasurde) [reply lib/ansible/modules/cloud/vmware/vmware_guest_disk_info.py:233]: This change is required due to https://github.com/ansible/ansible/commit/34868f07d5b32083d03cc4954287355b3b2f647e#commitcomment-35271563

**Comment by Akasurde** [comment]
Add a change log entry for this change [here](https://github.com/ansible/ansible/tree/devel/changelogs/fragments)

**Comment by goneri** [comment]
Hi!

Thank you for your PR. Since March 2, 2020, the Ansible team does not accept any new changes on the VMware modules on the core `ansible/ansible` repository.
The development continue but in the new VMware collection repository: https://github.com/ansible-collections/vmware
Could you please reopen the PR there?

Links:
- https://groups.google.com/forum/#!msg/ansible-devel/6UAfogEDtG8/wMTgxBGWCgAJ
- https://github.com/ansible-collections/overview/blob/master/README.rst

**Comment by goneri** [comment]
I've manually migrated your PR here, feel free to take it over: https://github.com/ansible-collections/vmware/pull/15

I close this one since it will never be merged.

**Pull request closed without merging**